frontend/src/app/App.tsx

```diff
@@ -143,7 +143,14 @@ const App = () => {
               <Route
                 path="/"
                 render={() => {
-                  return <Redirect to="/queue" />;
+                  return (
+                    <Redirect
```

Comment:
This should work but will lose the hash if we start to use it. To preserve everything including the hash you should be able to do something like:
```jsx
<Route
    path="/"
    exact
    render={({ location }) => (
       <Redirect to={{ ...location, pathname: "/queue" }} />
    )}
 />
```

Reply:
This is much better. updated.